lib/bn.js

```diff
@@ -459,6 +465,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
+  assert(typeof BN._Buffer !== 'undefined');
```

Comment:
What should be the variable name? Ideas:
- _Buffer or __Buffer
- nodeBuffer
- nodejsBuffer
- BufferCons
- BufferAPI

I think I would go with nodeBuffer or nodejsBuffer.